examples/nextjs/pages/_app.js

```diff
@@ -29,5 +30,6 @@ export default function MyApp(props) {
 
 MyApp.propTypes = {
   Component: PropTypes.elementType.isRequired,
+  emotionCache: PropTypes.object.isRequired,
```

Comment:
We do have a default cache so it isn't required with PropTypes semantics since `undefined` is allowed on the client, no?